qutebrowser/components/adblock.py

```diff
@@ -147,7 +178,9 @@ def _read_hosts_file(self, filename: str, target: typing.Set[str]) -> bool:
         try:
             with open(filename, 'r', encoding='utf-8') as f:
                 for line in f:
-                    target.add(line.strip())
+                    for host in self._read_hosts_line(line):
```

Comment:
Can this be `target.update(self._read_hosts_line(line))`?

set update can be used below as well to save the loop